addons/website/models/ir_actions.py

```diff
@@ -2,64 +2,60 @@
 
 import urlparse
 
+from openerp import api, fields, models
 from openerp.http import request
-from openerp.osv import fields, osv
 
 
-class actions_server(osv.Model):
+class ActionsServer(models.Model):
     """ Add website option in server actions. """
     _name = 'ir.actions.server'
     _inherit = ['ir.actions.server']
 
-    def _compute_website_url(self, cr, uid, id, website_path, xml_id, context=None):
-        base_url = self.pool['ir.config_parameter'].get_param(cr, uid, 'web.base.url', context=context)
-        link = website_path or xml_id or (id and '%d' % id) or ''
+    @api.multi
+    def _compute_website_url(self):
+        self.ensure_one()
```

Comment:
why ensure one ? couldn't you compute this field for more than one server action?